### PR TITLE
[#150180771] Fixed errors in present documentation

### PR DESCRIPTION
**Pull request by christian-casecommons**
https://www.pivotaltracker.com/story/show/150180771

Brief (see comments below):
 - Updated repository uris for intake-api and intake-app to use @pemacc private account
 - Added section for creating and publishing ruby image
 - Moved publish squid image to web-proxy section
 - Updated kms secrets creation for jenkins

### CODE REVIEW

File: content/intake-accelerator/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The Intake Accelerator Docker image needs to be published to the **casecommons/i
 1\. Clone the Intake application from the following forked repository (https://github.com/mixja/intake) to your local environment and ensure you checkout the **production_workflow** branch:
 
 {{< highlight make "hl_lines=9" >}}
-$ git clone git@github.com:mixja/intake.git
+$ git clone git@github.com:pgeyleg/intake.git
```
Comment (christian-casecommons) [inline content/intake-accelerator/index.md]: changed to pema private

File: content/intake-api/index.md [hunk 2]
```diff
@@ -344,7 +458,7 @@ Now that we have built the Intake base Docker image, we can build and publish th
 1\. Clone the Intake API application to your local environment and ensure you checkout the **workflow_enhancements** branch:
 
 {{< highlight make "hl_lines=9" >}}
-$ git clone git@github.com:ca-cwds/intake_api.git
+$ git clone git@github.com:pgeyleg/intake_api.git
```
Comment (christian-casecommons) [inline content/intake-api/index.md]: change to pema private

File: content/intake-api/index.md [hunk 3]
```diff
@@ -118,6 +118,120 @@ TASK [aws-cloudformation : configure application stack] ************************
 
 6\.  Note that because we are validating two domain names (the primary `*.demo.cloudhotspot.co` subject name and the `demo.cloudhotspot.co` subject alternative name), you must approve separate validation requests for each domain.
 
+## Creating a Ruby Image
```
Comment (christian-casecommons) [inline content/intake-api/index.md]: Added section covering ruby image publishing

File: content/web-proxy/index.md [hunk 4]
```diff
@@ -152,6 +152,111 @@ $ mv ~/Downloads/admin.pem.txt ~/.ssh/demo-resources-admin.pem
 $ chmod 600 ~/.ssh/demo-resources-admin.pem
 ```
 
+## Creating the Proxy Image
```
Comment (christian-casecommons) [inline content/web-proxy/index.md]: Moved proxy image publishing to web-proxy section

File: content/jenkins/index.md [hunk 5]
```diff
@@ -361,7 +361,7 @@ config_worker_docker_image: specifies the Docker image used to run the Jenkins m
 
 ```bash
 $ export AWS_PROFILE=demo-resources-admin
-$ aws kms encrypt --key-id 11710b57-c424-4df7-ab3d-20358820edd9 --plaintext $(openssl rand -base64 32)
+$ aws kms encrypt --key-id 11710b57-c424-4df7-ab3d-20358820edd9 --plaintext "password"
```
Comment (christian-casecommons) [inline content/jenkins/index.md]: Changed to cleartext since this is the password deployed container will use

Comment (pemacc) [reply content/jenkins/index.md]: $(openssl rand -base64 32) generates text which is secure. I am fine with keeping it as it is. We should be able to decrypt from KMS if we need to see the plain password.

**Pull request closed without merging**